src/common/helpers/github-url.js

```diff
@@ -1,6 +1,16 @@
-const getGitHubRepoUrl = (owner, name) => {
+import parseGitHubUrl from 'parse-github-url';
+
+export const getGitHubRepoUrl = (owner, name) => {
   let repository = !name ? owner : `${owner}/${name}`;
   return `https://github.com/${repository}`;
 };
 
-export default getGitHubRepoUrl;
+export const parseGitHubRepoUrl = (url) => {
```

Comment:
Moved out from repository input reducer.

Now instead of just returning full repo name it returns whole object with owner, name, fullName and url, so in any place we need those we can get these in same way.